.reuse/dep5

```diff
@@ -1,5 +1,5 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
-Files: .github/pull_request_template.md .github/ISSUE_TEMPLATE/*.md
+Files: .github/pull_request_template.md .github/ISSUE_TEMPLATE/*.md *.json
```

Comment:
Excluding all json files may be too lax, can we exclude just specific files?

Reply:
We can't use comments in `json` AFAIK

Reply:
You can have a separate file named `filename.json.license` which contains the license info for `filename.json`.
See `nix/sources.json.license` as an example.

Reply:
Done